Sources/Common/Error.swift

```diff
@@ -65,6 +68,54 @@ public indirect enum OperaError: Error {
     case networking(error: Error, request: URLRequest?, response: HTTPURLResponse?, json: Any?)
     case parsing(error: Error, request: URLRequest?, response: HTTPURLResponse?, json: Any?)
 
+    public var error: Error {
```

Comment:
would be nice to implement `localizedDescription`, take a look to [AFError impl](https://github.com/Alamofire/Alamofire/blob/master/Source/AFError.swift)